src/css/our-lessons.css

```diff
@@ -0,0 +1,162 @@
+.our-lessons-description {
+  margin-bottom: 40px;
+}
+
+.our-lessons-item {
+  border-radius: 16px;
+  border: 1px solid var(--green);
+  background-color: var(--white);
+  margin-bottom: 40px;
+  padding: 40px 20px;
+  color: var(--soft-black);
+  max-width: 336px;
+}
+
+.our-lessons-item:nth-child(2) {
+  border-radius: 16px;
+  background-color: var(--green);
+  margin-bottom: 40px;
+  padding: 40px 20px;
+  color: var(--white);
+  max-width: 336px;
+}
+
+.our-lessons-item-title {
+  font-weight: 700;
+  line-height: 1.2;
+  letter-spacing: -0.02em;
+}
+
+.our-lessons-title-a {
+  margin-bottom: 16px;
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 1.2;
+  letter-spacing: -0.02em;
+}
+
+.our-lessons-title-price {
+  margin-bottom: 40px;
+  font-size: 32px;
+}
+
+.black-color {
+  color: var(--black);
+}
+
+.our-lessons-item-description {
+  max-width: 291px;
+  min-height: 132px;
+  margin-bottom: 20px;
+  font-size: 14px;
+  line-height: 1.71;
+  letter-spacing: -0.02em;
+  text-align: center;
+}
+
+.our-lessons-button {
+  padding: 14px 76px;
+  border: 1px solid rgba(255, 255, 255, 0.4);
+  min-width: 256px;
+  border-radius: 12px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 1.71;
+  color: var(--white);
+  background-color: var(--green);
+  transition: background-color var(--animation-fast);
+}
+
+.our-lessons-button:hover {
+  background-color: #38ae5f;
+}
+
+.our-lessons-secondary-button {
+  padding: 14px 76px;
+  border: 1px solid rgba(255, 255, 255, 0.4);
```

Comment:
check if this color exists in vars
add one if it doesn't

Reply:
done!